specs/mrtcore/PrimaryLanguageOverride/PrimaryLanguageOverride.md

```diff
@@ -0,0 +1,206 @@
+
+```c#
+public static string PrimaryLanguageOverride { get; set; }
+```
+
+# Property Value
+
+[String](https://learn.microsoft.com/dotnet/api/system.string)
+
+A [BCP-47 language tag](https://tools.ietf.org/html/bcp47). The app can set it to override the
+language. It must be a single language tag; a delimited list of language tags will result in default
+language being used.
+
+The PrimaryLanguageOverride property should only be set to languages available for the app.
+
+When your app gets the value, PrimaryLanguageOverride returns either a single language tag (if your
+app has previously set the property) or an empty string.
+
+# Remarks
+
+Apps normally run with language settings determined by the system by comparing the languages
+supported by the app with the language preferences of the user. The PrimaryLanguageOverride property
+is used to override that behavior by setting a single specific language.
+
+The PrimaryLanguageOverride setting is not persisted between sessions. It should be set each time
+the app is loaded. You should set PrimaryLanguageOverride setting in early stage of app loading,
+before any resource is loaded. It should only be set based on user input presented in settings UI.
+The property can be read at any time. If the property has never been set, it returns an empty
+string.
+
+When you set the PrimaryLanguageOverride, this will be immediately reflected in resources loaded
+afterwards, both resources loaded from the code and the resources loaded by XAML. However, this
+change may not take effect immediately on resources already loaded in the app UI. To make sure the
+app responds to such changes, actions may be needed to reload resources. Those requirements may vary
+depending on the UI framework used by the app, and it may be necessary to restart the app.
+
+This property (Microsoft.Windows.ApplicationModel.Resources.PrimaryLanguageOverride) takes
+precedence over the existing one
+[Windows.Globalization.ApplicationLanguages.PrimaryLanguageOverride](https://learn.microsoft.com/uwp/api/windows.globalization.applicationlanguages.primarylanguageoverride).
+
+Example:
+
+```c#
+public partial class App : Application
+{
+    // ...
+    public App()
+    {
+        // Set the primary language override to German (Germany)
+        // This will cause the app to use the German (Germany) resources
+        // Both resources loaded manually with ResourceLoader and XAML resources fetched with x:Uid
+        // will be German (Germany)
+        Microsoft.Windows.ApplicationModel.Resources.ApplicationLanguages.PrimaryLanguageOverride = "de-DE";
+
+        Microsoft.Windows.ApplicationModel.Resources.ResourceLoader resourceLoader = new Microsoft.Windows.ApplicationModel.Resources.ResourceLoader();
+        var resourceString = resourceLoader.GetString("SampleString");
+
+        this.InitializeComponent();
+    }
+    // ...
+}
+```
+
+## ApplicationLanguages.GetLanguagesForUser(User) Method
```

Comment:
I think MUA is not applicable to WinAppSDK apps, so I think we can actually remove this from this type.

Remove the api here and then update the Background to state that it is being intentionally left off due to not being relevant for WinAppSDK.